ciri/README.md

```diff
@@ -45,9 +45,16 @@ Command line options
 
 Long option | Short option | Description | Example input
 --- | --- | --- | ---
-`--help` | `-h` | Displays the usage |
-`--log-level` | `-l` | Log level amongst: "debug", "info", "notice", "warning", "error", "critical", "alert" and "emergency" | `-l debug`
-`--neighbors` | `-n` | Neighbors that you are connected with will be added via this option | `-n "udp://148.148.148.148:14265 udp://[2001:db8:a0b:12f0::1]:14265"`
-`--udp-receiver-port` | `-u` | UDP receiver port | `-u 14600`
-`--tcp-receiver-port` | `-t` | TCP receiver port | `-t 15600`
-`--testnet` | | Makes it possible to run IRI with the IOTA testnet | `--testnet`
+`--help` | `-h` | Displays the usage. |
+`--log-level` | `-l` | Log level amongst: "debug", "info", "notice", "warning", "error", "critical", "alert" and "emergency". | `-l debug`
+`--mwm`||The minimum weight magnitude is the number of trailing 0s that must appear in the end of a transaction hash. Increasing this number by 1 will result in proof of work that is 3 times as hard.| `--mwm 14`
```

Comment:
Number of trailing ternary 0s that must appear at the end of a transaction hash. Difficulty can be described as 3^mwm